packages/charts/src/components/CarbonChart/CarbonChart.tsx

```diff
@@ -0,0 +1,216 @@
+import * as React from 'react';
+
+import {
+  ThemeInterface,
+  ThemeContext,
+  useIsInverse,
+  styled,
+} from 'react-magma-dom';
+
+import {
+  AreaChart,
+  SimpleBarChart,
+  BubbleChart,
+  WordCloudChart,
+} from '@carbon/charts-react';
+import '@carbon/styles/css/styles.css';
+import '@carbon/charts/styles.css';
+
+export enum CarbonChartType {
+  area = 'area',
+  bar = 'bar',
+  bubble = 'bubble',
+  word = 'word',
+}
+
+export interface CarbonChartProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  isInverse?: boolean;
+  /**
+   * @internal
+   */
+  theme?: ThemeInterface;
+  type: CarbonChartType;
+  dataSet: Array<Object>;
+  options: Object;
+}
+
+const CarbonChartWrapper = styled.div<{
+  isInverse?: boolean;
+  theme: ThemeInterface;
+}>`
+
+  .cds--cc--tooltip .content-box .datapoint-tooltip p{
+    font-size:14px;
+    padding:4px;
+  }
+
+  .cds--modal-container{
+    border-radius:${props => props.theme.borderRadius};
+  }
+
+  p, div, text, .cds--cc--axes g.axis .axis-title,
+  .cds--cc--title p.title,
+  .cds--cc--axes g.axis g.tick text {
+    font-family: 'Work Sans', Helvetica, sans-serif !important;
```

Comment:
Could we update all these values to use our theme variables? Font size/family, margin, etc etc